module/docs/entra-powershell-beta/Applications/Grant-EntraBetaMcpServerPermission.md

```diff
@@ -0,0 +1,259 @@
+---
+author: giomachar
+description: This article provides details on the Grant-EntraBetaMcpServerPermission command.
+external help file: Microsoft.Entra.Beta.Applications-Help.xml
+Locale: en-US
+Module Name: Microsoft.Entra.Beta.Applications
+ms.author: gmachar
+ms.date: 11/06/2024
+ms.topic: reference
+online version: https://learn.microsoft.com/powershell/module/Microsoft.Entra.Beta.Applications/Grant-EntraBetaMcpServerPermission
+schema: 2.0.0
+title: Grant-EntraBetaMcpServerPermission
+---
+
+# Grant-EntraBetaMcpServerPermission
+
+## SYNOPSIS
+
+Grants delegated permissions to a Model Context Protocol (MCP) client for Microsoft MCP Server for Enterprise.
+
+## SYNTAX
+
+### PredefinedClient (Default)
+```powershell
+Grant-EntraBetaMcpServerPermission
+ -MCPClient <String>
+ [<CommonParameters>]
+```
+
+### CustomClient
+```powershell
+Grant-EntraBetaMcpServerPermission
+ -MCPClientServicePrincipalId <String>
+ [<CommonParameters>]
+```
+
+### PredefinedClientScopes
+```powershell
+Grant-EntraBetaMcpServerPermission
+ -MCPClient <String>
+ -Scopes <String[]>
+ [<CommonParameters>]
+```
+
+### CustomClientScopes
+```powershell
+Grant-EntraBetaMcpServerPermission
+ -MCPClientServicePrincipalId <String>
+ -Scopes <String[]>
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Grant-EntraBetaMcpServerPermission` cmdlet grants delegated permissions to a Model Context Protocol (MCP) client for accessing the Microsoft MCP Server for Enterprise. This cmdlet can work with a predefined MCP client (Visual Studio Code, Visual Studio, ChatGPT, or Claude Desktop) or a custom MCP client specified by its service principal ID.
+
+The cmdlet creates an OAuth2 permission grant that allows the specified MCP client to access the Microsoft MCP Server for Enterprise on behalf of users. You can grant all available scopes or specify particular scopes to grant. The cmdlet returns an OAuth2PermissionGrant object that conforms to the Microsoft Graph API resource specification.
+
+For delegated scenarios, the calling user needs at least one of the following Microsoft Entra roles:
+- Global Administrator
```

Comment:
I suggest we remove the `Global Administrator` role as we have lesser privileged roles available. This is in line with SFI guidelines on using Global Administrator.
```suggestion
```